js/count_monkeys.test.js

```diff
@@ -10,14 +10,16 @@ const functionsToTest = [
   monkeyCountDoWhile,
 ];
 
-functionsToTest.forEach((functionsToTest) => {
-  describe(`count_monkeys ${functionsToTest} function`, () => {
+const namesArray = ['for loop', 'from', 'do while'];
+
+functionsToTest.forEach((functionsToTest, i) => {
```

Comment:
I just noticed, you are using the same `functionsToTest` for the array of functions, as well as the individual function being tested.

This would have been caught by a linter. Please invest some time in setting in up, so I don't have to point out issues like this one :smile:

Reply:
I've changed to .name solution and will look at this case.

Reply:
Sure 😄 Just rename the function parameter name 😄